README.md

```diff
@@ -1,30 +1,26 @@
-# Front-End Repo for Node/React Practicum
+## Description
 
-This will be the front-end for your team's practicum project.
+The Music App is a React application where users can stream music, add songs to wishlist and select and buy songs.
 
-It is suggested that you run these instructions **after** you setup the back-end server first.
-You can go through these steps during your first group meeting in case you need assistance from your mentors.
 
-You will have two folders inside one team folder (one for front-end and one for back-end). Name the parent folder something appropriate (in the below example we title it "Practicum Project"). Then clone directly (do not fork and clone) the front and back repos while inside the parent ("Practicum Project") project folder.
+## Screenshots
 
-![folders](images/folder_structure.png)
+![Signin](images\signin.png)
 
-> The front-end app (React) will be running on port 3000. The back-end server will be running on port 8000. You will need to run both the front-end app and the back-end server at the same time to test your app.
+![Home](images\homeapp)
 
-### Setting up local development environment
+## Technology Stack
 
-1. Clone this repository to the folder that was already created for both the front-end and back-end repos
-2. Run `npm install` to install dependencies
-3. Pull the latest version of the `main` branch (when needed)
-4. Run `npm start` to start the development server
-5. Open http://localhost:3000 with your browser to see the data received the back-end server.
-6. Now you have your front-end and back-end running locally!
+The app is composed of
+**Material UI for a straighforward design structure
+**Stripe.js for processing payment information
 
-#### Running the front-end server in Visual Studio Code
+## Contributors
 
-Note: In the below example, the group's front-end repository was named `bb-practicum-team1-front` and the back-end repository was named `bb-practicum-team-1-back`. Your repository will have a different name, but the rest should look the same.
-![vsc running](images/front-end-running-vsc.png)
+Contributors for the development of The Music App are
+
+![Masooma Jafari](https://github.com/JafariM)
+![Raheim Grant](https://github.com/Rjjayy)
+![Alberto Reyes](https://github.com/dayofthetech)
```

Comment:
For links that aren't images we only need `[]` rather than `![]`: [https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links)

```suggestion
[Masooma Jafari](https://github.com/JafariM)
[Raheim Grant](https://github.com/Rjjayy)
[Alberto Reyes](https://github.com/dayofthetech)
```

It would also be good to add mentions to our later contributors `AlonaVladymyrovaTrinity`  and `Musab-Kara` :smile: